draft-ietf-emu-rfc7170bis.md

```diff
@@ -446,27 +446,52 @@ allows support for protection of the peer's identity when using TLS
 client authentication.  An example of the exchanges using TLS
 renegotiation to protect privacy is shown in Appendix C.
 
-## Server Certificate Validation
+## Certificate creation and Validation
 
 As part of the TLS negotiation, the server presents a certificate to
-the peer.  The peer SHOULD verify the validity of the EAP server
-certificate and SHOULD also examine the EAP server name presented in
-the certificate in order to determine whether the EAP server can be
-trusted.  When performing server certificate validation,
-implementations MUST provide support for the rules in {{RFC5280}} for
-validating certificates against a known trust anchor.  In addition,
-implementations MUST support matching the realm portion of the peer's
-NAI against a SubjectAltName of type dNSName within the server
-certificate.  However, in certain deployments, this might not be
-turned on.  Please note that in the case where the EAP authentication
+the peer.
+This section describes the validation that the Peer SHOULD do in order to accept the certificate.
+There are a number of situations where the Peer does not do validation:
+
+* when the Peer has the Server's End Entity (EE) certificate pinned or loaded directly into it's Trusted Anchor store {{?RFC4949}}.
+
+* when the Peer is certain that it will be using an authenticated Inner method
+
+Situations in which the Peer will be doing some kind of onboarding operation ("bootstrapping") may require that the Peer defer the verification of the digital signature on the server's certificate, however the content verification described below MUST still be performed.
+
+For the server certificate validation, implementations MUST provide support for the rules in {{RFC5280}} for validating certificates against a known trust anchor.
+
+The Peer MUST verify the validity of the EAP server certificate and MUST also examine the EAP server name presented in the certificate in order to determine whether the EAP server can be
+trusted.
+
+The Peer MUST do DNS-ID style verification of the Realm portion of the Peer's NAI.
+{{!ID.ietf-uta-rfc6125bis, Section 6}} describes the mechanism in detail, and this document notes that comparison against the SubjectDN is no longer indicated.
```

Comment:
Can we have a normative reference to an I-D?

I'm wary of delaying this document until 6125bis is published.  Can we not just depend on 6215?

Plus, no implementation today does this DNS-ID matching.  The peer doesn't have network access, so it can't do DNS checks.  And it's accessing the server by WiFi SSID (or wired port), not by hostname.  Is there a more generic way of configuring this?

Reply:
> Can we have a normative reference to an I-D?
> 
> I'm wary of delaying this document until 6125bis is published. Can we not just depend on 6215?

No, the IESG will just tell you to rewrite and update.
6125bis is in the RFC-editor Q already.

> Plus, no implementation today does this DNS-ID matching. The peer doesn't have network access, so it can't do DNS checks. And it's accessing the server by WiFi SSID (or wired port), not by hostname. Is there a more generic way of configuring this?

DNS-ID *style* does not imply DNS.
It means if you wrote "foo.example" in the configuration file as the NAI, then it had better have foo.example in the subjectAldName.

Reply:
That's good to know.  I'll update it to reference 6125bis.